docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -6,14 +8,110 @@ The parent key for all of the following parameters is `cilium`.
 
 [horizontal]
 type:: string
-default:: `syn-cilium`
+default:: `cilium`
 
 The namespace in which to deploy this component.
 
+== `install_method`
+
+[horizontal]
+type:: string
+default:: `helm`
+possible values:: `helm`, `olm`
+
+The installation method for Cilium.
+`olm` uses the https://docs.openshift.com/container-platform/4.1/applications/operators/olm-understanding-olm.html[Openshift Operator Lifecycle Manager].
```

Comment:
```suggestion
`olm` uses the https://docs.openshift.com/container-platform/4.9/operators/understanding/olm/olm-understanding-olm.html[Openshift Operator Lifecycle Manager].
```

Reply:
Thanks for this one. I did not find the current doc...